pallets/ethereum-transactions/src/ethereum_transaction.rs

```diff
@@ -306,16 +306,10 @@ impl EthAbiHelper {
     }
 
     pub fn generate_ethereum_description_for_signature_request(
```

Comment:
Probably rename this function. I think what it is actually doing is `encode_arguments`.

Reply:
This function is no longer used anywhere, so it is to be deleted